src/v3/src/util/getEventContext.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { IdxTransaction } from '@okta/okta-auth-js';
+
+import { EventContext } from '../../../types';
+import { getV1ClassName } from '../../../v2/ion/ViewClassNamesFactory';
+
+export const getEventContext = (transaction: IdxTransaction): EventContext => {
+  const formName = transaction.nextStep?.name;
+  const authenticatorKey = transaction.context.currentAuthenticator.value.key;
```

Comment:
seems like `currentAuthenticator` is miss typed in authjs, add `?` after it to avoid runtime error